src/dialogs/domain-toggler/dialog-domain-toggler.ts

```diff
@@ -37,7 +37,11 @@ class DomainTogglerDialog extends LitElement {
         opened
         @opened-changed=${this._openedChanged}
       >
-        <h2>Toggle Domains</h2>
+        <h2>
+          ${this.hass.localize(
+            "ui.panel.config.entity_registry.toggle_domains"
```

Comment:
This is not the `entity_registry` make a section like `ui.dialogs.domain-toggler`

Reply:
Done with https://github.com/home-assistant/home-assistant-polymer/pull/4141/commits/4a400b8fcaa3998bdb2a4c8b0350b963d124fa64